### ISSUE TITLE
Insufficient contrast on sign in button with small text

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#4858]
The white text inside of the grey box for the sign in also doesn’t pass the contrast guidelines for small text. It is however high enough for big text.
